src/js/msp/MSPHelper.js

```diff
@@ -2579,33 +2585,34 @@ MspHelper.prototype.sendLedStripConfig = function(onCompleteCallback) {
 
     function send_next_led_strip_config() {
 
-        var led = FC.LED_STRIP[ledIndex];
-        var ledDirectionLetters =        ['n', 'e', 's', 'w', 'u', 'd'];      // in LSB bit order
-        var ledFunctionLetters =         ['i', 'w', 'f', 'a', 't', 'r', 'c', 'g', 's', 'b', 'l']; // in LSB bit order
-        var ledBaseFunctionLetters =     ['c', 'f', 'a', 'l', 's', 'g', 'r']; // in LSB bit
+        const buffer = [];
+
+        const led = FC.LED_STRIP[ledIndex];
+        const ledDirectionLetters =        ['n', 'e', 's', 'w', 'u', 'd'];      // in LSB bit order
```

Comment:
This is actually repeated from https://github.com/betaflight/betaflight-configurator/pull/2308/files#diff-2a7a24b74c7564918c438782f9a53a977205c0c79fd016a069b6dca546563adfR1202 - should be de-duplicated.